share/examples/bitcoin.conf

```diff
@@ -63,7 +63,12 @@
 # Port on which to listen for connections (default: 8333, testnet: 18333, signet: 38333, regtest: 18444)
 #port=
 
-# Maximum number of inbound+outbound connections.
+# Maximum number of inbound + outbound connections (default: 125). This option
+# applies only if inbound connections are enabled, otherwise the number of connections
+# will not be more than 11 (8 full-relay connections, 2 block-relay-only ones, and
+# occasionally 1 short-lived feeler or extra outbound block-relay-only connection.
```

Comment:
missing closing parentheses
```suggestion
# occasionally 1 short-lived feeler or extra outbound block-relay-only connection).
```

Reply:
Thanks! sorry, was just re-pushing a change. Fixing by replacing the opening one with a colon.